src/main/java/jenkins/authentication/tokens/api/AuthenticationTokens.java

```diff
@@ -129,7 +136,14 @@ private AuthenticationTokens() {
         // we want the best match first
         SortedMap<Integer,AuthenticationTokenSource> matches = new TreeMap<Integer, AuthenticationTokenSource>(
                 Collections.reverseOrder());
-        for (AuthenticationTokenSource<?, ?> source : Jenkins.getInstance()
+        Jenkins jenkins = Jenkins.getInstance();
+        if(jenkins == null){
+            LogRecord lr = new LogRecord(Level.FINE,
+                "No Jenkins object was found; no conversion could be made.");
+            LOGGER.log(lr);
+            return null;
+        }
```

Comment:
@kwhetstone Did you forget to address this in your [last commit](https://github.com/jenkinsci/authentication-tokens-plugin/pull/4/commits/b485c9acdd317559ac58676d9abee41b4f441229)?